toffy/fov_watcher.py

```diff
@@ -179,16 +188,21 @@ def on_created(self, event: FileCreatedEvent):
 
             logf.write(
                 f'{datetime.now().strftime("%d/%m/%Y %H:%M:%S")} -- '
-                f'Extracting {point_name}'
+                f'Extracting {point_name}\n'
             )
 
             # run per_fov callbacks
             for fov_func in self.per_fov:
                 logf.write(
                     f'{datetime.now().strftime("%d/%m/%Y %H:%M:%S")} -- '
-                    f'Running {fov_func.__name__} on {point_name}'
+                    f'Running {fov_func.__name__} on {point_name}\n'
                 )
-                fov_func(self.run_folder, point_name, self.watcher_out)
+
+                callback_dir = os.path.join(self.watcher_out, fov_func.__name__.split('_')[0])
```

Comment:
I think we'll want the option for different base paths for the different callbacks.

Reply:
Sure.  Right now, if you pass in `D:\\some_place` into `start_watcher`, it will create the subfolder `D:\\some_place\\run_name_XXXX` and put each callback's output into it a respective subfolder within there. In these terms do you want `D:\\some_place` to be configurable 'per-callback'?